common/app/views/fragments/img.scala.html

```diff
@@ -41,6 +41,8 @@
 
         @if(amp) {
             <meta itemprop="url" content="@ImgSrc.getFallbackUrl(picture)">
+            <meta itemprop="width" content="@ImgSrc.getFallbackAsset(picture).fold(0)(_.width)">
+            <meta itemprop="height" content="@ImgSrc.getFallbackAsset(picture).fold(0)(_.height)">
```

Comment:
I don't think image metadata needs to be amp specific but since `url` was already in the `if` I went with the flow ;)